src/sql/base/browser/ui/chart/chart.component.ts

```diff
@@ -0,0 +1,163 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { Component, Inject, forwardRef, ChangeDetectorRef } from '@angular/core';
+import { ChartOptions } from 'chart.js';
+import { } from 'ng2-charts';
+import { Disposable } from 'vs/base/common/lifecycle';
+
+export interface ChartData {
```

Comment:
After making any changes from the comments above, make sure you aren't duplicating type definitions with what's left over. Since this is just being used as a type you can just use the stuff from azdata directly - no need to redefine it here.

Reply:
Fixed this